lib/adbc_driver.ex

```diff
@@ -54,18 +54,21 @@ defmodule Adbc.Driver do
     end
   end
 
-  defp do_download(url, triplet, ignore_proxy, driver_name) do
-    with {:ok, zip_data} <- Helper.download(url, ignore_proxy),
-         {:ok, zip_handle} <- :zip.zip_open(zip_data, [:cooked, :memory]) do
-      for {:ok, zip_files} <- [:zip.table(zip_data, [:cooked])],
+  defp do_download(url, triplet, version, ignore_proxy, driver_name) do
+    with {:ok, adbc_cache_dir} = adbc_cache_dir(),
+         cache_path = Path.join(adbc_cache_dir, "#{triplet}-#{version}.zip"),
+         :ok <- Helper.download(url, ignore_proxy, cache_path),
+         cache_path = String.to_charlist(cache_path),
```

Comment:
I would move the cache dir handling inside `Helper.download` so this is something like

```suggestion
    with {:ok, driver_path} <- Helper.download(url, ignore_proxy, cache_path),
```

WDYT?

Reply:
Yeah, I think we can move it to `Adbc.Helper`/